server/jwt_test.go

```diff
@@ -4497,9 +4497,9 @@ func TestJWTAccountOps(t *testing.T) {
 		t.Run("", func(t *testing.T) {
 			var syspub, sysjwt, sysCreds string
 			createAccountAndUser(&syspub, &sysjwt, &sysCreds)
+			defer os.Remove(sysCreds)
```

Comment:
should use the https://golang.org/pkg/testing/#T.TempDir

Reply:
not going to change this as this will drag though to a lot of utility functions that use ioutil.tempfile...